<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrigPictureProcessWindow</class>
 <widget class="QWidget" name="OrigPictureProcessWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>558</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>新宋体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="u_widget_top1" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton
{
background-color:rgba(255,255,255,1);
border:1px solid rgba(207,214,226,1);
border-radius:3px;
}
QToolButton::hover
{
background-color:rgba(229,243,255,1);
border:1px solid rgba(207,214,226,1);
border-radius:3px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnUndo">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;返回上一步，最多可以返回5步&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/app.qrc">
          <normaloff>:/Images/fileReview/icos6.ico</normaloff>:/Images/fileReview/icos6.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnRedo">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;下一步&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/app.qrc">
          <normaloff>:/Images/fileReview/icos7.ico</normaloff>:/Images/fileReview/icos7.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnDrag">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;拖拽&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/app.qrc">
          <normaloff>:/Images/fileReview/icos18.ico</normaloff>:/Images/fileReview/icos18.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnMannualChangeAngle">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;纠偏&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;自动纠偏&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/app.qrc">
          <normaloff>:/Images/fileReview/icos17.ico</normaloff>:/Images/fileReview/icos17.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnCutBorder">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;切边&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/app.qrc">
          <normaloff>:/Images/fileReview/icos9.ico</normaloff>:/Images/fileReview/icos9.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnAutoCorrectAngle">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;自动纠偏，点击后图片自动纠偏完成&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/app.qrc">
          <normaloff>:/Images/fileReview/icos0.ico</normaloff>:/Images/fileReview/icos0.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnBatchAutoChangAngle">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;批量纠偏&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>批量自动纠偏</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/app.qrc">
          <normaloff>:/Images/fileReview/icos0.ico</normaloff>:/Images/fileReview/icos0.ico</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnPickPhoto">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;抠取照片&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/app.qrc">
          <normaloff>:/Images/fileReview/icos11.ico</normaloff>:/Images/fileReview/icos11.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnPickPhoto2">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;抠取照片到基本信息&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/app.qrc">
          <normaloff>:/Images/fileReview/ico65.png</normaloff>:/Images/fileReview/ico65.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnClearImage">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;转优化图像&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/app.qrc">
          <normaloff>:/Images/fileReview/ico11.ico</normaloff>:/Images/fileReview/ico11.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnPartClear">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;对局部图像进行优化&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>局部优化</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/app.qrc">
          <normaloff>:/Images/fileReview/ico11.ico</normaloff>:/Images/fileReview/ico11.ico</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnShowMore">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>其他功能</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/app.qrc">
          <normaloff>:/Images/fileReview/icos3.ico</normaloff>:/Images/fileReview/icos3.ico</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSave">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>保存图片</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/app.qrc">
          <normaloff>:/Images/fileReview/icos15.ico</normaloff>:/Images/fileReview/icos15.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSet">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>快捷键设置</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/app.qrc">
          <normaloff>:/qss/Images/fileReview/icos26.png</normaloff>:/qss/Images/fileReview/icos26.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnTip">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>图像操作技巧</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/app.qrc">
          <normaloff>:/Images/fileReview/icos28.png</normaloff>:/Images/fileReview/icos28.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_top2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton
{
background-color:rgba(255,255,255,1);
border:1px solid rgba(207,214,226,1);
border-radius:3px;
}
QToolButton::hover
{
background-color:rgba(229,243,255,1);
border:1px solid rgba(207,214,226,1);
border-radius:3px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnAICorrectAngle">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>AI纠偏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnAIBatCorrectAngle">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>AI批量纠偏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnAICutBorder1">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>AI切边(1)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnAIBatCutBorder1">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>AI批量切边(1)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnAICutBorder2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>AI切边(2)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnAIBatCutBorder2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>AI批量切边(2)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnAIRemoveStain">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>AI转高清</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnAIBatRemoveStain">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>AI批量转高清</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>361</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="u_widget_top2" native="true">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="chkShowRuler">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>标尺</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkScrollBarShow">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>滚动条</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkLocker">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>锁定操作</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="grpGamma" native="true">
        <property name="minimumSize">
         <size>
          <width>370</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>370</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2x">
           <property name="text">
            <string>阈值：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QThresholdLineEdit" name="edtThreshold">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;双击查看直方图,请输入0到255之间的数字&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3x">
           <property name="text">
            <string>Gamma值：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtGamma">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;请输入0到9之间的数字&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkAutoThresholdValue">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>自动取阈值</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="u_widget_PictureProcessWindow" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QThresholdLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">src/control/editcontrol/qthresholdlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../res/app.qrc"/>
 </resources>
 <connections/>
</ui>
